src/www/ui/api/index.php

```diff
@@ -246,11 +246,11 @@ function (\Slim\Routing\RouteCollectorProxy $app) {
 ////////////////////////////ADMIN-USERS/////////////////////
 $app->group('/users',
   function (\Slim\Routing\RouteCollectorProxy $app) {
-    $app->get('[/{id:\\d+}]', UserController::class . ':getUsers');
-    $app->put('[/{id:\\d+}]', UserController::class . ':updateUser');
-    $app->post('', UserController::class . ':addUser');
-    $app->delete('/{id:\\d+}', UserController::class . ':deleteUser');
     $app->get('/self', UserController::class . ':getCurrentUser');
+    $app->get('[/{pathParam:[\\w\\- \\.\\d]+}]', UserController::class . ':getUsers');
+    $app->put('[/{pathParam:[\\w\\- \\.\\d]+}]', UserController::class . ':updateUser');
+    $app->post('', UserController::class . ':addUser');
+    $app->delete('/{pathParam:[\\w\\- \\.\\d]+}', UserController::class . ':deleteUser');
```

Comment:
I would rather not allow whitespace in the group/user name. Please remove the whitespace from the regex.

Reply:
By doing this, we won't be able to access `Default User` via requests.

Reply:
Hey @GMishx. Please guide on the above.

Reply:
Humm... We can rename the user to `defaultUser` or something like that. Allowing spaces will only cause problems.

Reply:
Done